taxi/urls.py

```diff
@@ -16,8 +22,26 @@
         ManufacturerListView.as_view(),
         name="manufacturer-list",
     ),
+    path(
+        "manufacturers/create/",
+        ManufacturerCreateView.as_view(),
+        name="manufacturer-create",
+    ),
+    path(
+        "manufacturers/<int:pk>/update/",
+        ManufacturerUpdateView.as_view(),
+        name="manufacturer-update",
+    ),
+    path(
+        "manufacturers/<int:pk>/delete/",
+        ManufacturerDeleteView.as_view(),
+        name="manufacturer-delete",
+    ),
```

Comment:
There is no URL pattern for the `ManufacturerDetailView`. You should add a path for the manufacturer detail view, similar to how it's done for cars and drivers, to allow access to individual manufacturer details.